kaolin/transforms/pointcloudfunc.py

```diff
@@ -25,6 +25,52 @@
 EPS = 1e-6
 
 
+def shift(cloud: Union[torch.Tensor, PointCloud],
+          shf: Union[float, int, torch.Tensor],
+          inplace: Optional[bool] = True):
+    """Shift the input pointcloud by a shift factor.
+
+    Args:
+        cloud (torch.Tensor or kaolin.rep.PointCloud): pointcloud (ndims >= 2).
+        shf (float, int, torch.Tensor): shift factor (scaler, or tensor).
+        inplace (bool, optional): Bool to make the transform in-place
+    
+    Returns:
+        (torch.Tensor): shifted pointcloud pf the same shape as input.
+    
+    Shape:
+        - cloud: :math:`(B x N x D)` (or) :math:`(N x D)`, where :math:`(B)`
+            is the batchsize, :math:`(N)` is the number of points per cloud,
+            and :math:`(D)` is the dimensionality of each cloud.
+        - shf: :math:`(1)` or :math:`(B)`.
+
+    Example:
+        >>> points = torch.rand(1000,3)
+        >>> points2 = shift(points, torch.FloatTensor([3]))
+    """
+
+    if isinstance(cloud, np.ndarray):
+        cloud = torch.from_numpy(cloud)
+    
+    if isinstance(shf, np.ndarray):
+        shf = torch.from_numpy(shf)
+    
+    if isinstance(cloud, PointCloud):
+        cloud = cloud.points
+    
+    if isinstance(shf, int) or isinstance(shf, float):
+        shf = torch.Tensor([shf]).to(cloud.device)
```

Comment:
nice catch! thanks for pointing out.

Reply:
My note was actually not a strong indication for the PR (more like a general advice)

I believe converting shf to torch.Tensor is not useful as https://github.com/pushkalkatara/kaolin/blob/translatePC/kaolin/transforms/pointcloudfunc.py#L60 would work just as fine with python `int` and `float`. It would work regardless of `cloud.device`.